Source/IO/ROOTSpectrogramWriter/KTROOTSpectrogramWriter.hh

```diff
@@ -72,14 +71,19 @@ namespace Katydid
             template< typename XDataType >
             void AddPowerSpectralDensityDataCoreHelper(Nymph::KTDataPtr data, DataTypeBundle& dataBundle);
 
+            template< typename XDataType >
+            void AddAggregatePowerSpectrumDataCoreHelper(Nymph::KTDataPtr data, DataTypeBundle& dataBundle);
```

Comment:
Why did you need to add new helper functions?  These appear to be the same as the non-aggregated ones.

Reply:
This was a repetition in code that was caused because of a conflict with the develop branch. I have removed the repeated code. Fixed in commit ae569a85bc54b10279dfe0c08acefb98f8c59bd2.